### ISSUE TITLE
Releasing a workspace does not update local path dependencies

### ISSUE THREAD

**Issue by sebschmi** [crate-ci/cargo-release#223]
I have a workspace with multiple crates that depend on each other, with the dependency specified via a path. Before release, I had the following dependencies of the `genome-graph` crate:

```toml
[dependencies]
bigraph = {path = "../bigraph", version = "0.1.0-alpha.4"}
compact-genome = {path = "../compact-genome", version = "0.1.0-alpha.4"}
num-traits = "0.2"
bio = "0.31"
error-chain = "0.12"
```

The version of `bigraph` in the workspace was `0.1.0-alpha.4`.
I then executed `cargo release alpha` on the workspace root, which bumped the versions of all crates and released them as expected. But it did **NOT** bump the versions in the `[dependencies]` sections. As a result, the released `genome-graph` crate has version `0.1.0-alpha.5`, but still points to `bigraph` version `0.1.0-alpha.4`, which is wrong.

How do I get `cargo-release` to bump also the versions in the declaractions of `[dependencies]`?

**Comment by epage** [comment]
For anyone else  wanting more details, the crate is at https://github.com/algbio/practical-omnitigs/tree/master/implementation

**Comment by epage** [comment]
The only thing I can think of is https://github.com/sunng87/cargo-release/issues/206 where we update dependents in a workspace when doing the regular version bump but then don't during the post-release version bump.

Does that look like your issue?

**Comment by sebschmi** [comment]
Thank you for the quick answer.
That looks similar. I am currently at cargo-release 0.13.4. Does updating to 0.13.5 include the pull request (#209 ) mentioned in #206 ?

**Comment by epage** [comment]
Hmm, looks like #209 was included in 0.13.4

**Comment by sebschmi** [comment]
Hm, I just executed `cargo release rc`, using cargo-release version 0.13.5. It again forgot to update the versions.

The dependencies are still at alpha, even though I released the whole workspace at once. See here for example: https://crates.io/crates/genome-graph/0.1.0-rc.1
________________________________

**Comment by sebschmi** [comment]
Ah, so what's happening seems to be that the dependencies within the workspace are updated only after release, and not before. But they would need to be updated before release, such that the release crates depend on the correct versions.
________________________________

**Comment by sebschmi** [comment]
And - when I fix the version numbers before executing `cargo release`, it complains that the given versions don't exist. Hm, so no way to release with correct dependencies at the moment except manually. It would really be super nice and help me a lot if that would get fixed!
________________________________

**Comment by tylerhawkes** [comment]
I'm also running into this right now

**Comment by vadixidav** [comment]
Hitting this issue on an internal project. Many workspaces are interdependent, so this seems pretty crucial. Temporarily I am just going to remove the versions.

**Comment by epage** [comment]
#295 resolved #206.

If people are running a version with #295, could you include the output with logging including trace?  Otherwise, there is not enough information here to root cause.

**Comment by epage** [comment]
Goling ahead and closing
